specification/storage/data-plane/Microsoft.QueueStorage/stable/2018-03-28/queue.json

```diff
@@ -486,6 +486,7 @@
               },
               "x-ms-approximate-messages-count": {
                 "type": "integer",
+                "format": "int64",
```

Comment:
This will cause a break in client libraries for strongly typed languages, but we can easily version around it the same way we did for Jumbo Blobs.